src/dashboards/cluster-analysis.json

```diff
@@ -1,14 +1,5 @@
 {
```

Comment:
What do these changes do?

Reply:
Not sure why you are asking. There is no formatting changes, only actual changes.